.github/workflows/test_validate.yaml

```diff
@@ -0,0 +1,18 @@
+name: Validate tests
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v3
+        with:
+          cache: pip
+          cache-dependency-path: bin/requirements.txt
+      - name: Validate tests
+        run: "ls grammar/*.json functions/*.json | xargs -I {} jsonschema test_schema.json -i {}"
```

Comment:
Does this run without error even if the `functions` folder does not exist?
Did you intend to have this PR be a follow up of #52 ?